test_runtime.sh

```diff
@@ -77,9 +77,9 @@ trap cleanup EXIT
 tar -xf  rootfs.tar.gz -C ${TESTDIR}
 cp runtimetest ${TESTDIR}
 
-ocitools generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
+ocitools generate --tty --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
```

Comment:
If docker-runc failed without --tty. I think that's docker-runc's problem. The spec says /dev/console is the default device for Linux which must be supplied by runtime.